src/main/java/de/cas/mse/exercise/SumDigPower.java

```diff
@@ -38,4 +24,34 @@
 
 	}
 
+    private long doHappyNumberCalculation(long happyNumberCandidate) {
+        List<Integer> singleDigits = toListOfDigits(happyNumberCandidate);
+        long sum = calculateSum(singleDigits);
+        return sum;
+    }
+
+    private long calculateSum(List<Integer> singleDigits) {
+        long sum = 0;
+        for (int index = 0; index < singleDigits.size(); index++) {
```

Comment:
We like the way you extracted the exponent, maybe it would be more readable if we name the index according to its digit.